evalml/tests/conftest.py

```diff
@@ -237,6 +237,20 @@ class MockBinaryClassificationPipeline(BinaryClassificationPipeline):
     return MockBinaryClassificationPipeline
 
 
```

Comment:
This had to be done to prevent the patching of .score() to be detached during the mapping process.  This *may* be a reasonable way to test the parallel stuff.